app/src/main/java/com/example/checkpoint/MainActivity.kt

```diff
@@ -268,7 +272,7 @@ fun BottomSheetContentSmall() {
 fun BottomSheetContentLarge() {
     Text(text = "Bottom Sheet Content Large",
         modifier = Modifier.padding(16.dp),
-        style = MaterialTheme.typography.h6,
+        style = MaterialTheme.typography.h4,
```

Comment:
Expanding the text as the user draws up their view might be a good idea. so when the bottom sheet scaffold is expanded the text is slightly larger.